.github/workflows/codeql.yml

```
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "release/**", "feature/**" ]
+  pull_request:
+    branches: [ "staging/docs-6" ]
```

I think we might as well structure this how we use it, with doubling up of PR and branch for staging/release. It's not like scanning will run twice if a commit matches both triggers.

```suggestion
  push:
    branches: [ "feature/**", "hotfix/**", "staging/**", "release/**"  ]
  pull_request:
    branches: [ "staging/**", "release/**" ]
```